esp-hal/src/spi/master.rs

```diff
@@ -951,6 +951,13 @@ mod dma {
         Mode,
     };
 
+    fn empty_rx_buf() -> DmaRxBuf {
+        unwrap!(DmaRxBuf::new(&mut [], &mut []))
```

Comment:
Could we make this a public method on `DmaRx|TxBuf`? I've wanted this as an end-user before :D. Maybe it could be `const` too?

Reply:
It can't be const, &mut [] won't compile unfortunately.

Options are, which one would you prefer?
- `DmaRxTxBuf::empty()`
- `derive(Default)`

Reply:
`DmaRxTxBuf::empty()` is my preference.